Base/Implementations/Validation.cs

```diff
@@ -115,6 +121,7 @@ public void Validate()
                         _errorLevel = _errorLevel | _RuleMap[s].ErrorLevel;
                     }
                     _Errors.AddRange(_RuleMap[s].Errors);
+                    _ErrorMessages.AddRange(_RuleMap[s].ErrorMessages);
```

Comment:
you are always adding and never clearing.  you probably need a method near line 109 _ErrorMessages.Clear() or _ErrorMessages = new list<ErrorMessage>;